pymc/tests/test_variational_inference.py

```diff
@@ -831,11 +851,13 @@ def aevb_model():
         pm.Normal("y", size=(2,))
     x = model.x
     y = model.y
-    mu = aesara.shared(x.init_value)
-    rho = aesara.shared(np.zeros_like(x.init_value))
+    xr = model.initial_values[model.rvs_to_values[x]]
+    mu = aesara.shared(xr)
+    rho = aesara.shared(np.zeros_like(xr))
```

Comment:
This won't work after #4983 because the `model.initial_values` entry for the `HalfFlat` will be `"moment"`.
What are you trying to achieve with these shared variables?